packages/amplify-graphql-api-construct-tests/src/commands.ts

```diff
@@ -52,7 +52,7 @@ export type InitCDKProjectProps = {
  * @returns a promise which resolves to the stack name
  */
 export const initCDKProject = async (cwd: string, templatePath: string, props?: InitCDKProjectProps): Promise<string> => {
-  const { cdkVersion = '2.80.0', additionalDependencies = [] } = props ?? {};
+  const { cdkVersion = '2.97.0', additionalDependencies = [] } = props ?? {};
```

Comment:
I see you noted in the description that a bump is necessary due to the unsupported Node14 Lambda runtime. Why only bump to `2.97.0` when the current version is [2.128.0](https://github.com/aws/aws-cdk/releases/tag/v2.128.0)?

Reply:
Another excellent question. We have an ongoing conversation with the CLI team about what version we should actually be testing:
- The CDK version officially supported by Gen1 CLI is 2.80.0 (which fails as noted)
- The earliest CDK version we can safely use in our CDK constructs is 2.97.0
- The version officially supported by the Gen2 CLI was [2.110.0](https://github.com/aws-amplify/amplify-backend/blame/a01f6b9777f4fe7394b05de115f37e70c10d9af8/packages/backend/package.json) up until 2 days ago, when the team bumped it to [2.127.0](https://github.com/aws-amplify/amplify-backend/blob/main/packages/backend/package.json#L38)

For now, we'll maintain the earliest working version so we introduce the fewest changes. We'll synchronize the supported versions between the CDK construct, the Gen2 CLI, and the associated tests, which we're tracking in a backlog item elsewhere.

Reply:
Thanks for the context.